test/fixtures.py

```diff
@@ -64,6 +64,20 @@ def run_once(self):
         self.rebound_add_one.run_once()
 
 
+class MultipleRebindsFragment(ExpFragment):
+    def build_fragment(self):
+        self.fragments = []
+        for idx in range(10):
+            frag = self.setattr_fragment(f"add_one_{idx}", AddOneFragment)
+            self.fragments.append(frag)
+        additional_owners = [(frag, ) for frag in self.fragments[1:]]
+        self.setattr_param_rebind("value", self.fragments[0], None, additional_owners)
```

Comment:
Should change the default of the rebound parameter here to a different number, e.g. `1.0`, and check if **all** subfragments return `2.0` instead of `1.0`.